datadog_checks_dev/datadog_checks/dev/tooling/create.py

```diff
@@ -29,7 +29,7 @@ def get_valid_templates():
 
 def construct_template_fields(integration_name, repo_choice, **kwargs):
     normalized_integration_name = normalize_package_name(integration_name)
-    check_name_kebab = re.sub('_| ', '-', integration_name.lower())
+    check_name_kebab = re.sub('_', '-', integration_name.lower())
```

Comment:
This looks like an unrelated behavioral change to me, as we wouldn't be replacing spaces with `-` now?

Reply:
Oops, ty